code/modules/antagonists/clockcult/clock_structures/traps/power_null.dm

```diff
@@ -0,0 +1,17 @@
+/obj/structure/destructible/clockwork/trap/power_nullifier
+	name = "power nullifier"
+	desc = "A well hidden set of wires and connections."
+	clockwork_desc = "When triggered, it sets off a 3x3 emp pulse with the center suffering a stronger pulse."
+	icon_state = "electric_trap"
+	break_message = "<span class='warning'>The power nullifier sparks, then slowly crumbles to debris!</span>"
+	max_integrity = 40
+	density = FALSE
+	var/activated = FALSE
+
+/obj/structure/destructible/clockwork/trap/power_nullifier/activate()
+	if(!activated)
+		activated = TRUE
+		empulse(get_turf(src),1,1,TRUE)
```

Comment:
"it's balanced" he says, knowing full well people will spam these to make energy weapons completely useless.

Reply:
Well maybe they treat EMP traps like brass spears and destroy them before they get close.

It's not like the EMP traps have legs. Basically if get crew to destroy the traps before they blindly walk in with energy weapons, then they can be destroyed.

I mean you get the exact same result if you walk into a brass skewer carrying an energy weapon: You drop it and usually someone with a high IQ steals it. I've done this plenty of times as a drone. With this in mind, it's actually worse to emp energy weapons as both sides can't use it.